src/home_robot_hw/home_robot_hw/env/stretch_abstract_env.py

```diff
@@ -434,7 +434,7 @@ def process_depth(self, depth):
         depth[depth > self.max_depth_val] = MAX_DEPTH_REPLACEMENT_VALUE
         return depth
 
-    def get_images(self, compute_xyz=False, rotate_images=True):
+    def get_images(self, compute_xyz=False, rotate_images=False):
```

Comment:
Can we just remove the `rotate_images` argument?

Reply:
we need to verify that it's correct by default for this use case; hopefully we can remove it. does get images even need the flag still?

Reply:
has this been tested on the robot? can Vidhi run the habitat ovmm code, or can you, before we merge?